conventions/services/utils.py

```diff
@@ -183,3 +188,43 @@ def _normalize(numero: str | None) -> str | None:
     parts.append(datetime.now().strftime("%Y-%m-%d_%H-%M"))
 
     return f"{'_'.join(parts)}.pdf"
+
+
+def _can_delete_action_alerte(alerte, destinataire):
+    if destinataire == "MO":
+        for dest in alerte["destinataires"]:
+            if dest["codeProfil"] == "MO_PERS_MORALE":
+                return True
+
+    if destinataire == "SG":
+        for dest in alerte["destinataires"]:
+            if dest["codeProfil"] == "SER_GEST":
+                return True
```

Comment:
il y a peut-être moyen de l'écrire comme ça : 
```suggestion
        return any(dest["codeProfil"] == "SER_GEST" for dest in alerte["destinataires"])
```